server/src/main/java/org/elasticsearch/index/mapper/RootObjectMapper.java

```diff
@@ -106,19 +110,54 @@ public RootObjectMapper.Builder addRuntimeFields(Map<String, RuntimeField> runti
 
         @Override
         public RootObjectMapper build(MapperBuilderContext context) {
+            Map<String, Mapper> mappers = buildMappers(context);
+            mappers.putAll(getAliasMappers(mappers, context));
```

Comment:
I think we want the aliases to be visible, but maybe I misunderstood the question. What would be the alternative here?

Reply:
I thought that the intention was to have the passthrough mapper, and field aliases hidden somehow in the resulting mappings. Maybe I misunderstood.

Reply:
I don't believe that the alias field mappers would be visible in the mappings. That's because the get mapping request doesn't return a serialized version of the `RootObjectMapper`. Instead, we're returning the `source` as a blob, as it has been provided in the put mapping request.